core/server/data/migrations/versions/2.13/1-remove-empty-strings.js

```diff
@@ -0,0 +1,55 @@
+const Promise = require('bluebird');
+const common = require('../../../../lib/common');
+const schema = require('../../../schema');
+
+/*
+ * [{
+ *   tableName: 'posts',
+ *   columns: ['custom_excerpt', 'description', 'etc...']
+ * }]
+ * */
+const tablesToUpdate = Object.keys(schema.tables).reduce((tablesToUpdate, tableName) => {
+    const table = schema.tables[tableName];
+    const columns = Object.keys(table).filter(columnName => table[columnName].nullable);
+    if (!columns.length) {
+        return tablesToUpdate;
+    }
+    return tablesToUpdate.concat({
+        tableName,
+        columns
+    });
+}, []);
+
+const createReplace = (connection, from, to) => (tableName, columnName) => {
+    common.logging.info(
+        `Updating ${tableName}, setting all '${from}' in ${columnName} to ${to}`
+    );
+
+    return connection(tableName)
```

Comment:
I'm thinking do we have to handle the case of missing tables/columns - I'm thinking it's possible to have this migration run when the schema is different to what's in the db.

Reply:
Good thinking. Ideally, we should not interrupt migration in case there is a mismatch between the schema and the actual db. This could be a case on any migration actually, but here we are of touching >100 fields, so maybe some basic messaging on which field was skipped due to mismatch would be helpful.